client/app/components/Parameters.less

```diff
@@ -0,0 +1,121 @@
+@import '../assets/less/ant';
+
+.drag-handle {
+  background: linear-gradient(90deg, transparent 0px, white 1px, white 2px)
+              center,
+              linear-gradient(transparent 0px, white 1px, white 2px) center, #111111;
+  background-size: 2px 2px;
+  display: inline-block;
+  width: 6px;
+  height: 36px;
+  vertical-align: bottom;
+  margin-right: 5px;
+  cursor: move;
+}
+
+.parameter-block {
+  display: inline-block;
+  background: white;
+  padding: 3px 6px 6px;
+
+  .parameter-container[data-draggable] & {
+    margin: 4px 0 0 4px;
+  }
+
+  &.parameter-dragged {
+    box-shadow:  0 4px 9px -3px rgba(102, 136, 153, 0.15);
+  }
```

Comment:
```suggestion
    width: auto !important;
  }
```

Or else this happens:
<img src=https://user-images.githubusercontent.com/486954/62290883-08951880-b46b-11e9-9025-6c137af2ed83.gif width=200 />

That's cause in this case the element computes to `width: 93.2px` but the sortable rounds it down to `93px`. So keeping it untouched with `width: auto !important` fixes it .